web/themes/joinup/prototype/sass/components/_sidebar-menu.sass

```diff
@@ -42,8 +42,7 @@
   text-decoration: none
   &:hover,
   &:active,
-  &:focus,
-  &.is-active
```

Comment:
Restored. Maybe in the future we should think about ticket for designer to prepare pdf style guide how items should behave in different states.